release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -975,6 +975,13 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-15-image-registry-bug-fixes"]
 ==== Image Registry
 
+* Previously, the Image Registry pruner relied on a cluster role that was managed by the openshift-apiserver. This could cause the pruner job to intermittently fail during an upgrade. Now, the Image Registry Operator is responsible for creating the pruner cluster role, which resolves the issue. (link:https://issues.redhat.com/browse/OCPBUGS-18969[*OCPBUGS-18969*])
```

Comment:
```suggestion
* Previously, the Image Registry pruner relied on a cluster role that was managed by the `openshift-apiserver` service. This could cause the pruner job to intermittently fail during an upgrade. Now, the Image Registry Operator is responsible for creating the pruner cluster role, which resolves the issue. (link:https://issues.redhat.com/browse/OCPBUGS-18969[*OCPBUGS-18969*])
```
?
or maybe just OpenShift API server?